src/index.js

```diff
@@ -0,0 +1,84 @@
+import './style.css';
+import menuImgSrc from './menu.png';
+import enterImgSrc from './enter.png';
+import refreshImgSrc from './refresh.png';
+
+const body = document.getElementsByTagName('body')[0];
+const container = document.querySelector('.to-do-list');
+
+const h1Container = document.querySelector('.h1-container');
+const refreshImg = document.createElement('img');
+refreshImg.src = refreshImgSrc;
+refreshImg.className = 'refresh-img';
+h1Container.appendChild(refreshImg);
+const form = document.createElement('form');
+const input = document.createElement('input');
+input.type = 'text';
+input.name = 'description';
+input.placeholder = 'Add to your list...';
+const inputLine = document.createElement('div');
+inputLine.className = 'input-line';
+inputLine.appendChild(input);
+const enterImg = document.createElement('img');
+enterImg.src = enterImgSrc;
+enterImg.className = 'enter-img';
+
+inputLine.appendChild(enterImg);
+form.appendChild(inputLine);
+container.appendChild(form);
+
+const arryOfTasks = [];
+
+const Task = {
+  create(description, index) {
+    const task = Object.create(this);
+    task.description = description;
+    task.completed = false;
+    task.index = index;
+    return task;
+  },
+};
+
+input.addEventListener('keydown', (e) => {
+  if (e.key === 'Enter' && input.value !== '') {
+    e.preventDefault();
+    const task = Task.create(input.value, arryOfTasks.length);
+    arryOfTasks.push(task);
+    input.value = '';
+    localStorage.setItem('tasks', JSON.stringify(arryOfTasks));
+  }
+});
```

Comment:
- [ ] I really appreciate your effort in adding the "add" functionality but this milestone doesn't require you to implement that you just need to create a temporary array of todo task objects and render them by iterating.
- Requirement:
  ![image](https://user-images.githubusercontent.com/104968236/235934318-e3521abc-f775-4add-bb68-f2ea81ec2418.png)
- [ ] Please revisit the [requirements here](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md)